prac_06/guitar.py

```diff
@@ -0,0 +1,16 @@
+class Guitar:
+    def __init__(self, name="", year=0, cost=0):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def get_age(self):
+        current_year = 2023
+        age = current_year - self.year
```

Comment:
Instead of defining a variable here you can just return current_year - self.year